### PR TITLE
Data fetching

### PR DESCRIPTION
**Pull request by Sita22**
- the api was tested with console.log (it works!)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
 import logo from "./logo.svg";
 import "./App.css";
+import { useEffect, useState } from "react";
 
 function App() {
+  const [data, setData] = useState(null)
+  useEffect(() => {
+    async function loadData() {
+      const response = await fetch('https://redi-final-restaurants.herokuapp.com/restaurants')
```
Comment (Firasso) [inline src/App.js:9]: why not extract the link into  a constant

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+      const data = await response.json()
+      setData(data)
+    }
+    loadData()
+  }, [])
+  console.log(data)
```
Comment (Firasso) [inline src/App.js:15]: let remove the console log

**Review by Firasso — APPROVED** [review]
you can ignore my example  comments :D good job 👏

**Comment by OdirahNze** [comment]
I have seen this and noted this.

**Review by OdirahNze — APPROVED** [review]

**Review by Amjadoudeh — COMMENTED** [review]
nice work

**Pull request merged**